modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2RoleClaimTestCase.java

```diff
@@ -172,112 +186,51 @@ public void testSendAuthorizedPostAfterRoleUpdate() throws Exception {
-
-        Claim roleClaim = new Claim();
-        roleClaim.setClaimUri(ROLES_CLAIM_URI);
-        ClaimMapping roleClaimMapping = new ClaimMapping();
-        roleClaimMapping.setRequested(true);
-        roleClaimMapping.setLocalClaim(roleClaim);
-        roleClaimMapping.setRemoteClaim(roleClaim);
-
-        claimConfig.setClaimMappings(new org.wso2.carbon.identity.application.common.model.xsd
-                .ClaimMapping[]{emailClaimMapping, roleClaimMapping});
-
-        serviceProvider.setClaimConfig(claimConfig);
-        serviceProvider.setOutboundProvisioningConfig(new OutboundProvisioningConfig());
-        List<InboundAuthenticationRequestConfig> authRequestList =
-                new ArrayList<InboundAuthenticationRequestConfig>();
-
-        if (consumerKey != null) {
-            InboundAuthenticationRequestConfig opicAuthenticationRequest =
-                    new InboundAuthenticationRequestConfig();
-            opicAuthenticationRequest.setInboundAuthKey(consumerKey);
-            opicAuthenticationRequest.setInboundAuthType("oauth2");
-            if (consumerSecret != null && !consumerSecret.isEmpty()) {
-                Property property = new Property();
-                property.setName("oauthConsumerSecret");
-                property.setValue(consumerSecret);
-                Property[] properties = {property};
-                opicAuthenticationRequest.setProperties(properties);
-            }
-            authRequestList.add(opicAuthenticationRequest);
-        }
-
-        String passiveSTSRealm = SERVICE_PROVIDER_NAME;
-        if (passiveSTSRealm != null) {
-            InboundAuthenticationRequestConfig opicAuthenticationRequest =
-                    new InboundAuthenticationRequestConfig();
-            opicAuthenticationRequest.setInboundAuthKey(passiveSTSRealm);
-            opicAuthenticationRequest.setInboundAuthType("passivests");
-            authRequestList.add(opicAuthenticationRequest);
-        }
-
-        String openidRealm = SERVICE_PROVIDER_NAME;
-        if (openidRealm != null) {
-            InboundAuthenticationRequestConfig opicAuthenticationRequest =
-                    new InboundAuthenticationRequestConfig();
-            opicAuthenticationRequest.setInboundAuthKey(openidRealm);
-            opicAuthenticationRequest.setInboundAuthType("openid");
-            authRequestList.add(opicAuthenticationRequest);
-        }
-
-        if (authRequestList.size() > 0) {
-            serviceProvider.getInboundAuthenticationConfig()
-                    .setInboundAuthenticationRequestConfigs(authRequestList.toArray(new
-                            InboundAuthenticationRequestConfig[authRequestList.size()]));
-        }
-        appMgtclient.updateApplicationData(serviceProvider);
-        return appDtoResult;
+    private PatchOperationRequestObject getAddUserPatchRole(String userId) {
+        RoleItemAddGroupobj patchRoleItem = new RoleItemAddGroupobj();
```

Comment:
Shall we fix the formatting issue by adding the missing new line?

Reply:
Fixed with https://github.com/wso2/product-is/pull/20096